src/ape/api/networks.py

```diff
@@ -177,7 +177,7 @@ def serialize_transaction(self) -> bytes:
         if not self.signature:
             raise SignatureError("The transaction is not signed.")
 
-        txn_data = self.model_dump(exclude={"sender"})
+        txn_data = self.model_dump(exclude=("sender",))
```

Comment:
Is there a performance benefit to using tuple literal instead of set literal? Doing this somewhere else

Reply:
Tuples are the fastest collection-based data type in Python! You should only use sets if doing set stuff.

```
(ape08) ➜  ape git:(perf/pytest) python -m timeit "('foo',)"
50000000 loops, best of 5: 6.28 nsec per loop
(ape08) ➜  ape git:(perf/pytest) python -m timeit "{'foo'}" 
5000000 loops, best of 5: 59.4 nsec per loop
```

Reply:
btw I don't think `include=`/`exclude=` actually works with a tuple:
https://github.com/pydantic/pydantic/blob/a2f7271fd394e9d2a5b19cf41f531d7fb0a6b6f0/pydantic/deprecated/copy_internals.py#L24

Reply:
Odd:

```
>>> from pydantic import BaseModel
>>> class MyModel(BaseModel):
...    x: int = 3
...    y: int = 4
... 
>>> model = MyModel()
>>> model.model_dump(exclude=("x",))
{'y': 4}
```

I suppose since it is undocumented, I will undo it though, but I am going to open an issue in Pydantic.

Reply:
I reverted back to sets since tuple is undocumented, but we will see what happens with this: https://github.com/pydantic/pydantic/issues/10010

Reply:
There is some interesting discussion there, but in conclusions, yeah tuples are not yet properly supported, even though it works in all the cases we are using it. It sounds like they may add better support for any sequence types here in the future. Resolving this for now and sticking with sets!